public/js/index.js

```diff
@@ -1656,6 +1656,13 @@ $(function () {
 function showDeletePopup() {
     var $this = $(this);
     var filter = "";
+
+    // Add the repository name to class content
+    var pattern = /&([^]*)[^]gt;|<b>[^]*[^]<\/b>/;
```

Comment:
Could you explain what this is supposed to do? Since I think this does way more than you hope it will 🤔

Reply:
i use regex to get string `<strong>%s</strong>`  from the message delete repository. the message as follows.
`- This operation will permanently delete everything in the <strong>%s</strong> repository, including code, issues, comments, the wiki, and collaborators associations.`

then, will be replace with `<b>{repository_name}</b>` and it will be as follows
`- This operation will permanently delete everything in the <b>{repository_name}</b> repository, including code, issues, comments, the wiki, and collaborators associations.`

because  `<strong>` tag is not work i change with `<b>` tag, I think not a bad idea

Reply:
I think it should just replace `%s` and `%[1]s` without using additional regexp

Reply:
i tried it but not work

![scrot_2017-12-27--1514365900_721x235](https://user-images.githubusercontent.com/16787866/34376867-c7531328-eb20-11e7-844b-083c32c5daa9.png)

Reply:
`strong` will work, you need to add safe html when generating this dialog in golang template. Best option would be to add "{% repo_name %}" to $.i18n.Tr as second argument and than replace it

Reply:
If we want to use `<b>` instead of `<strong>`, we should just change the locale, instead of introducing client-side logic to change `<strong>` tags to `<b>`.

Also, won't this regexp match `<b>...</b>`, not `<strong>...</strong>`?

Reply:
@lafriks sorr, I don't understand what you mean.

Reply:
@ethantkoenig  i delete the regex and change from `b` tag to `strong`